src/setup/index.ts

```diff
@@ -36,27 +36,32 @@ const cwd = process.cwd();
     accountInfo = await got.post('https://faucet.altnet.rippletest.net/accounts', {json: true})
       .then(({body}) => body.account)
       .catch((err) => console.log('  ' + err.message));
+  } else if (serverUrl === 'wss://s.devnet.rippletest.net:51233') {
+    console.log('▲ Generating Devnet account credentials...');
+    accountInfo = await got.post('https://faucet.devnet.rippletest.net/accounts', {json: true})
+      .then(({body}) => body.account)
+      .catch((err) => console.log('  ' + err.message));
```

Comment:
Is it worth pulling this `await.then.catch` block into a function where you pass in `devnet` | `altnet`?